tools/boilerplate/boilerplate.go.txt

```diff
@@ -0,0 +1,4 @@
+// Copyright YEAR Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
```

Comment:
It would be nice if the boilerplate template ended with a blank line so a space exists before the `package` declaration in go files.